plugins/woocommerce-blocks/docs/third-party-developers/extensibility/checkout-block/how-checkout-processes-an-order.md

```diff
@@ -1,3 +1,9 @@
+---
+post_title: How The Checkout Block Processes an Order
```

Comment:
Based on https://github.com/woocommerce/woocommerce/blob/trunk/docs/quality-and-best-practices/grammar-punctuation-capitalization.md#capitalization, we should _"Use sentence case (not title case) for docs titles and subheadings."_ I suggest using sentence case for all docs titles to keep them consistent.

I also noticed that some docs titles are prefixed with `Cart and Checkout - ` while others are not. What are your thoughts on prefixing all or none, for consistency-reasons?